plugin-tray/lxqttray.cpp

```diff
@@ -373,11 +375,19 @@ void LXQtTray::stopTray()
 /************************************************
 
  ************************************************/
+void LXQtTray::onIconDestroyed(QObject * icon)
+{
+    mIcons.removeAll(qobject_cast<TrayIcon*>(icon));
```

Comment:
> Can you @RalfJung test with static_cast/reinterpret_cast?

Using `reinterpret_cast` is pretty much never the right thing to do. (I just noticed there are a bunch of them in lxqttray.cpp. That laves me very uneasy.^^)

Using `static_cast`, I am indeed not seeing any crashes anymore. Yay :D